salt/minion.py

```diff
@@ -1688,35 +1697,46 @@ def tune_in(self, start=True):
         ping_interval = self.opts.get('ping_interval', 0) * 60
         if ping_interval > 0:
             def ping_master():
-                self._fire_master('ping', 'minion_ping')
+                try:
+                    self._fire_master('ping', 'minion_ping')
+                except Exception:
```

Comment:
We should log the exception, since we're catching all Exceptions here. Otherwise it's hard to figure out why it failed.